src/components/enter-mfa/tests/enter-mfa-integration.test.ts

```diff
@@ -35,15 +36,18 @@ describe("Integration:: enter mfa", () => {
       .stub(sessionMiddleware, "validateSessionMiddleware")
       .callsFake(function (req: any, res: any, next: any): void {
         res.locals.sessionId = "tDy103saszhcxbQq0-mjdzU854";
-
         req.session.user = {
           email: "test@test.com",
           phoneNumber: PHONE_NUMBER,
           redactedPhoneNumber: PHONE_NUMBER,
+          reauthenticate: "12345",
```

Comment:
The only journey being tested is a reauth journey, worth testing for non-reauth journey to.